senderwatcher panic after unlock event

**Describe the bug**

Have to initialise the cache on node startup by calling `getSenderInfo` because `Reserve` will still be nil when receiving an `unlock` or `withdraw` event prior to anything else and we try to log `Reserve.FundsRemaining`. 

This is also useful for e.g. prometheus that is has access to the latest deposit and reserve values on node startup

``` 
******************************Eth Transaction******************************

Invoking transaction: "unlock". Inputs: ""  Hash: "0x7128d27577268cf3833117af0e92a910af5d46bea553f48ae294b39ef9b51e30".

***************************************************************************
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4ae35f3]

goroutine 63 [running]:
github.com/livepeer/go-livepeer/eth/watchers.(*SenderWatcher).handleLog(0xc0106c7770, 0xee22e310bfafb2b2, 0xb909a9b86e6efb7d, 0xba867639, 0xc0008de080, 0x2, 0x4, 0xc000874cc0, 0x40, 0x40, ...)
	/Users/nico/Desktop/livepeer/go-livepeer/eth/watchers/senderwatcher.go:266 +0x533
github.com/livepeer/go-livepeer/eth/watchers.(*SenderWatcher).handleBlockEvents(0xc0106c7770, 0xc000010678, 0x1, 0x1)
	/Users/nico/Desktop/livepeer/go-livepeer/eth/watchers/senderwatcher.go:149 +0x179
github.com/livepeer/go-livepeer/eth/watchers.(*SenderWatcher).Watch(0xc0106c7770)
	/Users/nico/Desktop/livepeer/go-livepeer/eth/watchers/senderwatcher.go:111 +0x485
created by main.main
	/Users/nico/Desktop/livepeer/go-livepeer/cmd/livepeer/livepeer.go:449 +0x2f19
```

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, safari]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

--- a/cmd/livepeer/livepeer.go
+++ b/cmd/livepeer/livepeer.go
@@ -52,8 +52,8 @@ var (
 	// The maximum blocks for the block watcher to retain
 	blockWatcherRetentionLimit = 20
 
-	// The gas required to redeem a PM ticket
-	redeemGas = 100000
+	// Estimate of the gas required to redeem a PM ticket
+	redeemGas = 250000
 	// The multiplier on the transaction cost to use for PM ticket faceValue
 	txCostMultiplier = 100
 

--- a/cmd/livepeer/livepeer.go
+++ b/cmd/livepeer/livepeer.go
@@ -586,6 +586,15 @@ func main() {
 				panic(fmt.Errorf("-depositMultiplier must be greater than 0, but %v provided. Restart the node with a valid value for -depositMultiplier", *depositMultiplier))
 			}
 
+			// Fetch and cache broadcaster on-chain info
+			info, err := senderWatcher.GetSenderInfo(n.Eth.Account().Address)
+			if err != nil {
+				glog.Error("Failed to get broadcaster on-chain info: ", err)
+				return
+			}
+			glog.Info("Broadcaster Deposit: ", eth.FormatUnits(info.Deposit, "ETH"))
+			glog.Info("Broadcaster Reserve: ", eth.FormatUnits(info.Reserve.FundsRemaining, "ETH"))
+
 			n.Sender = pm.NewSender(n.Eth, timeWatcher, senderWatcher, ev, *depositMultiplier)
 
 			if *pixelsPerUnit <= 0 {

--- a/eth/watchers/senderwatcher.go
+++ b/eth/watchers/senderwatcher.go
@@ -71,6 +71,11 @@ func (sw *SenderWatcher) setSenderInfo(addr ethcommon.Address, info *pm.SenderIn
 	sw.mu.Lock()
 	defer sw.mu.Unlock()
 	sw.senders[addr] = info
+
+	if addr == sw.lpEth.Account().Address && monitor.Enabled {
+		monitor.Deposit(addr.Hex(), info.Deposit)
+		monitor.Reserve(addr.Hex(), info.Reserve.FundsRemaining)
+	}
 }
 
 // ClaimedReserve returns the amount claimed from a sender's reserve by the node operator

--- a/install_ffmpeg.sh
+++ b/install_ffmpeg.sh
@@ -35,7 +35,7 @@ fi
 
 if [ ! -e "$HOME/nasm/nasm" ]; then
   # sudo apt-get -y install asciidoc xmlto # this fails :(
-  git clone -b nasm-2.14.02 https://repo.or.cz/nasm.git "$HOME/nasm"
+  git clone -b nasm-2.14.02 https://github.com/livepeer/nasm.git "$HOME/nasm"
   cd "$HOME/nasm"
   ./autogen.sh
   ./configure --prefix="$HOME/compiled"

